### PR TITLE
completed issue 1 creating an entry- Mohamed 

### PR DESCRIPTION
**Pull request by Mohamed-Elmessiry**
Screen Shot 2021-03-30 at 1 32 28 PM
Screen Shot 2021-03-30 at 1 32 37 PM

### CODE REVIEW

**Comment by Mohamed-Elmessiry** [comment]
Screen Shot 2021-03-30 at 2 06 02 PM
Screen Shot 2021-03-30 at 2 05 56 PM

**Comment by Mohamed-Elmessiry** [comment]
Screen Shot 2021-03-30 at 5 32 51 PM
Screen Shot 2021-03-30 at 5 31 29 PM

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  img {
+    width: 100%;
+    max-height: 400px;
+    object-fit: contain;
+  }
```
Comment (codingcodymiller) [inline css/styles.css]: This should just be the normal rule, not in the media query

File: js/data.js [hunk 2]
```diff
@@ -6,3 +6,16 @@ var data = {
   editing: null,
   nextEntryId: 1
 };
+
+var previousEntriesJson = localStorage.getItem('javascript-local-storage');
```
Comment (codingcodymiller) [inline js/data.js]: You should name this something like `code-journal-data`, not `javascript-local-storage`. Try to make the name describe the purpose of the data.

File: js/main.js [hunk 3]
```diff
@@ -1,2 +1,23 @@
...
+formSelector.addEventListener('submit', function (event) {
+  event.preventDefault();
+  var values = {
+    title: formSelector.elements.title.value,
+    url: formSelector.elements.url.value,
+    textbox: formSelector.elements.notes.value,
+    entryId: data.nextEntryId
+  };
+  data.nextEntryId++;
+  data.entries.push(values);
```
Comment (codingcodymiller) [inline js/main.js]: You should be adding to the beginning of the array, not the end. You will need to use `unshift` here.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**